bot/cogs/bot.py

```diff
@@ -60,6 +65,40 @@ def __init__(self, bot: AutoShardedBot):
 
         await ctx.invoke(self.info)
 
+    async def on_message(self, msg: Message):
+        if msg.channel.id in self.allowed:
+            if self.allowed[msg.channel.id]-time.time() > 300:
+                if msg.content.count("\n") >= 3:
+                    try:
+                        tree = ast.parse(msg.content)
+
+                        # Attempts to parse the message into an AST node.
+                        # Invalid Python code will raise a SyntaxError.
+                        if not all(isinstance(node, ast.Expr) for node in tree.body):
+
+                            # We don't want multiple lines of single words,
+                            # They would be syntactically valid Python but could also be
+                            # Just some random multiline text someone is sending.
+                            howto = """If you are going to post code in Discord, please use syntax highlighted blocks, as it makes it more legible for other users.
+
+To do this, you should input your content like this:
+
+\`\`\`python
+print("Hello world!")
+\`\`\`
+
+This will result in the following:
+```
+print("Hello world!")
+```
+"""
+                            information = Embed(title="Code formatting")
+                            information.add_field(name="Information", value=howto)
```

Comment:
Why not just `Embed(title="Code formatting", description=howto)`

Reply:
Because i did not know about that